docs/common-coreboot-docs/dasharo_tools_suite.md

```diff
@@ -2,30 +2,89 @@
 
 ## Introduction
 
-Dasharo Tools Suite (DTS) is a set of tools for deploying, updating and
-maintaining firmware on Dasharo supported devices. DTS is integrated into the
-firmware in Dasharo supported devices and can be used to update the firmware
-even when no OS is installed.
+Dasharo Tools Suite (DTS) is a set of tools running in a minimal Linux
+environment, with a goal of deploying, updating and maintaining firmware on
+Dasharo supported devices. For example, it can be used to update the firmware
+on a device, even when no OS is currently installed.
 
-## Requirements
+## Distribution methods
 
-- Dasharo device with DTS functionality integrated
-- Wired network connection
-- Secure Boot disabled
+It can be distributed in various ways. Currently, there are two distribution
+options:
 
-## Launching DTS
+* bootable USB stick image.
+* bootable over network (iPXE),
+
+## Flavors
+
+There is a common base, but there might be multiple flavors of the DTS images.
+Currently, there are:
+
+* CE
+    - `Dasharo HCL Report` - generates a package with logs containing hardware
+      information
+    - flashrom, fwupd, and many more useful tools
+    - can drop to shell to update the firmware manually
+* OEM
+    - on top of that, provides tools for automatic firmware deployment and
+      rollback (switching to Dasharo back and forth)
+
+## Bootable USB stick
+
+This documentation is compatible with the `v0.3.0` version of the DTS.
+
+### Requirements
+
+* USB stick (at least 1GB)
+* Wired network connection
+* Secure Boot disabled
+* [DTS CE v0.3.0 downloaded](https://cloud.3mdeb.com/index.php/s/Q7pAppm4gnRCef9/download)
+
+### Launching DTS
+
+* Flash the downloaded `dts-base-image-ce-v0.3.0.wic.gz` image onto USB stick.
+  - you can use cross-platform GUI installer - [Etcher](https://www.balena.io/etcher/)
+  - you can also use `dd` to flash from command line
+
+```bash
+gzip -cdk dts-base-image-ce-v0.3.0.wic.gz | sudo dd of=/dev/sdX bs=16M status=progress
+```
+
+* Insert the USB stick to a USB in your device
+
+* Boot from the USB stick
+
+### Using DTS
+
+* A menu should appear
+
+* Enter `1` to dump logs with hardware from your device
+
+![](../images/dts-hcl-run.png)
+
+## Bootable over network
+
+This documentation is compatible with the `v0.1.0` version of the DTS.
```

Comment:
There we go. Some parts of the document will be compatible with version X and some parts with version Y possibly causing confusion. Now I also think that it may be wise to show the most common usage here and link to more detailed documentation in the DTS repository. Documentation on DTS repository would be tagged along with the code and get rid of the problem of the versioned documentation here.

Reply:
I left it this way, becasue we have only two varianst in the wild at the moment:
- v0.1.0 - deployed on boot.3mdeb.com
- v0.3.0 - USB stick

With the next release, we can create a separate DTS documentation, which will be versioned for a given release.